docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -309,6 +309,25 @@ Serialization Formats
         encryption. Choose this unless you have explicit legacy compatibility
         requirements.
 
+.. class:: PublicFormat
+
+    .. versionadded:: 0.8
+
+    An enumeration for public key formats. Used with the ``public_bytes``
+    method available on
+    :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKeyWithSerialization`.
+
+    .. attribute:: SubjectPublicKeyInfo
```

Comment:
It's the name used by PyCrypto and [Crypto++](http://www.cryptopp.com/wiki/Keys_and_Formats#Public_Key_Format). Go calls it a "DER-encoded PKIX structure" but if you look in the source code it says "pkixPublicKey reflects a PKIX public key structure. See SubjectPublicKeyInfo in RFC 3280."